ibm/mas_devops/roles/aibroker/defaults/main.yml

```diff
@@ -108,5 +108,9 @@ mas_aibroker_db_secret_name: "{{ lookup('env', 'MAS_AIBROKER_DB_SECRET_NAME') |
 mas_aibroker_db_secret_key: "{{ lookup('env', 'MAS_AIBROKER_DB_SECRET_KEY') | default('password', true) }}"
 mas_aibroker_db_secret_value: "{{ lookup('env', 'MAS_AIBROKER_DB_SECRET_VALUE') | default('', true) }}"
 
-#aibroker Api
+# aibroker Api
 environment_type: "{{ lookup('env', 'ENVIRONMENT_TYPE') | default('non-production', true) }}"
+
+# cluster settings
+mas_aibroker_cluster_domain: "{{ lookup('env', 'MAS_AIBROKER_CLUSTER_DOMAIN') | default('', true) }}"
```

Comment:
MAS_AIBROKER_CLUSTER_DOMAIN needs to be documented in the role readme and the playbook readme